cluster-autoscaler/cloudprovider/aws/aws-sdk-go/aws/awserr/error.go

```diff
@@ -10,24 +10,23 @@ package awserr
 //
 // Example:
 //
-//     output, err := s3manage.Upload(svc, input, opts)
```

Comment:
nit: I think the go fmt in files other than `cluster-autoscaler/cloudprovider/gce/gce_price_info.go` probably doesn't block this PR.

It may be helpful to split this PR into two PRs-- one updating `cluster-autoscaler/cloudprovider/gce/gce_price_info.go`; the other one fixing go fmt in this repo

Reply:
The gofmt change was failing a presubmit test on the initial commit

Reply:
The error said:

> Please run hack/update-gofmt.sh to fix the following files:
./cluster-autoscaler/cloudprovider/gce/gce_price_info.go

There's no need to update all the other files.